include/titan/options.h

```diff
@@ -53,31 +63,37 @@ struct TitanCFOptions : public ColumnFamilyOptions {
   // Default: nullptr
   std::shared_ptr<Cache> blob_cache;
 
-  // Max batch size for gc
+  // Max batch size for GC.
   //
   // Default: 1GB
   uint64_t max_gc_batch_size{1 << 30};
 
-  // Min batch size for gc
+  // Min batch size for GC.
   //
   // Default: 512MB
   uint64_t min_gc_batch_size{512 << 20};
 
-  // The ratio of how much discardable size of a blob file can be GC
+  // The ratio of how much discardable size of a blob file can be GC.
   //
   // Default: 0.5
   float blob_file_discardable_ratio{0.5};
 
-  // The ratio of how much size of a blob file need to be sample before GC
+  // The ratio of how much size of a blob file need to be sample before GC.
   //
   // Default: 0.1
   float sample_file_size_ratio{0.1};
 
-  // The blob file size less than this option will be mark gc
+  // The blob file size less than this option will be mark GC.
   //
   // Default: 8MB
   uint64_t merge_small_file_threshold{8 << 20};
 
+  // The blob running mode used to turn off Titan. This option is only
+  // valid for default column family using key-value seperation.
```

Comment:
why is only valid for the default column?

Reply:
Non-Default CF do not adopt key-value separated approach for their value length are small. It's a upper level logic though, I shouldn't mention it here

Reply:
This constraint is only for TiKV. For other product, it may use other column families.